annoying/fields.py

```diff
@@ -65,6 +70,26 @@ class Page(models.Model):
         page.save()
     """
 
+    def __init__(self, serializer=None, deserializer=None, *args, **kwargs):
```

Comment:
I'm worried that this will break the default order of parameters on the JSONField. Is that the case?